examples/jsm/loaders/obj2/OBJLoader2Parser.js

```diff
@@ -337,17 +337,17 @@ OBJLoader2Parser.prototype = {
 		if ( this.logging.enabled ) console.time( 'OBJLoader2Parser.execute' );
 		this._configure();
 
-		let arrayBufferView = new Uint8Array( arrayBuffer );
+		const arrayBufferView = new Uint8Array( arrayBuffer );
 		this.contentRef = arrayBufferView;
-		let length = arrayBufferView.byteLength;
+		const length = arrayBufferView.byteLength;
 		this.globalCounts.totalBytes = length;
-		let buffer = new Array( 128 );
+		const buffer = new Array( 128 );
 
 		let bufferPointer = 0;
 		let slashesCount = 0;
 		let word = '';
 		let currentByte = 0;
-		for ( let code, currentByte = 0; currentByte < length; currentByte ++ ) {
+		for ( let code; currentByte < length; currentByte ++ ) {
```

Comment:
@kaisalmen I believe this was a bug. You have declared `currentByte` two times. Hence, the outer `currentByte` was always `0` which could potentially be a problem in the last call of `_processLine()`.

https://github.com/mrdoob/three.js/pull/20246/files#diff-0d91ee5a7e6fe5182c3ea5a4f747935eR387

Reply:
True, thank you for spotting this.